File-Formats/DocIO/Working-with-Table-Of-Contents.md

```diff
@@ -703,6 +715,562 @@ End Using
+        captionPara.AppendText(" " + picture.AlternativeText)
+        ' Apply formatting to the caption.
+        captionPara.ApplyStyle(BuiltinStyle.Caption)
+        captionPara.ParagraphFormat.BeforeSpacing = 8
+        captionPara.ParagraphFormat.HorizontalAlignment = HorizontalAlignment.Center
+    Next
+
+    ' Create a new paragraph.
+    paragraph = New WParagraph(document)
+    paragraph.AppendText("List of Tables")
+    ' Apply Heading1 style for paragraph.
+    paragraph.ApplyStyle(BuiltinStyle.Heading1)
+    ' Insert the paragraph.
+    document.LastSection.Body.ChildEntities.Insert(2, paragraph)
+
+    ' Create a new paragraph and append TOC.
+    paragraph = New WParagraph(document)
+    tableOfContent = paragraph.AppendTOC(1, 3)
+    ' Disable a flag to exclude heading style paragraphs in TOC entries.
+    tableOfContent.UseHeadingStyles = False
+    ' Set the name of SEQ field identifier for table of tables.
+    tableOfContent.TableOfFiguresLabel = "Table"
+    ' Insert the paragraph to the text body.
+    document.LastSection.Body.ChildEntities.Insert(3, paragraph)
+
+    ' Find all tables from the document.
+    Dim tables As List(Of Entity) = document.FindAllItemsByProperty(EntityType.Table, Nothing, Nothing)
+    ' Iterate each table and add caption.
+    For Each table As WTable In tables
+        ' Gets the table index.
+        Dim tableIndex As Integer = table.OwnerTextBody.ChildEntities.IndexOf(table)
+        ' Create a new paragraph and appends the sequence field to use as a caption.
+        Dim captionPara As New WParagraph(document)
+        captionPara.AppendText("Table ")
+        captionPara.AppendField("Table", FieldType.FieldSequence)
+        ' Set alternate text as caption for table.
+        captionPara.AppendText(" " + table.Description)
+        ' Apply formatting to the paragraph.
+        captionPara.ApplyStyle(BuiltinStyle.Caption)
+        captionPara.ParagraphFormat.BeforeSpacing = 8
+        captionPara.ParagraphFormat.HorizontalAlignment = HorizontalAlignment.Center
+        ' Insert the paragraph next to the table.
+        table.OwnerTextBody.ChildEntities.Insert(tableIndex + 1, captionPara)
+    Next
+
+    ' Update all document fields to update SEQ fields.
+    document.UpdateDocumentFields()
+    ' Update the table of contents.
+    document.UpdateTableOfContents()
+
+    ' Save the document
+    document.Save("Output.docx")
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from GitHub.
+
+By executing the program, you will get the **Word document** as follows.
+
+![Table of figures in Word document](WorkingWithTableOfContents/Table_of_Figures.png)
+
+### Exclude caption label and numbers
+
+The following code example illustrates how to exclude the caption's label and number in TOC entries during the creation of a table of figures using the IncludeCaptionLabelsAndNumbers API.
```

Comment:
The following code example shows how to add a table of figures by exclude the caption's label and number in Word document using IncludeCaptionLabelsAndNumbers API.